### PR TITLE
- Add Bitmark Upgrade Proposal (BUP).

### PR DESCRIPTION
**Pull request by araujobsd**
Issue: https://github.com/bitmark-inc/docs/issues/7

### CODE REVIEW

File: learning-bitmark/contributing-to-bitmark/bup.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## What is BUP?
+A Bitmark Upgrade Proposal (BUP) is a design document providing
+information to Bitmark community, or describing a new feature for
+Bitmark or its processes or environment. A BUP shall provide a
+concise technical specification of the feature and a rationale for the
+feature.
+
+## Where I can find the BUPs?
+We maintain an entity at Github only for BUP, we want to keep a clean
```
Comment (shannona) [inline learning-bitmark/contributing-to-bitmark/bup.md:9]: I assume you mean you maintain a repo?

File: learning-bitmark/contributing-to-bitmark/bup.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Bitmark or its processes or environment. A BUP shall provide a
+concise technical specification of the feature and a rationale for the
+feature.
+
+## Where I can find the BUPs?
+We maintain an entity at Github only for BUP, we want to keep a clean 
+environment where developers will not have distraction with other
+repos, this is the reason why we do not maintain it under
+Bitmark documentation ![repository](https://github.com/bitmark-inc/docs).
+
```
Comment (shannona) [inline learning-bitmark/contributing-to-bitmark/bup.md:13]: Not sure you need the additional explanation (this is the reason why we do not maintain it under
Bitmark documentation repository.)

File: learning-bitmark/contributing-to-bitmark/bup.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+To have full access to all BUPs, please access: ![Bitmark Upgrade Proposal](https://github.com/bitmark-inc/docs)
```
Comment (shannona) [inline learning-bitmark/contributing-to-bitmark/bup.md:14]: I assume you mean: https://github.com/bitmark-property-system/bups

**Review by shannona — COMMENTED** [review]
Short and to the point. Explains what BUPs are and where to find them.

**Comment by araujobsd** [comment]
@shannona same case as the another PR.

**Pull request merged**